### DISCUSSION TITLE
What's the development strategy here?

### DISCUSSION THREAD

**Discussion by alexgraul** [electric-sql/pglite#119] [General]
Interested in using this for a project but it would require porting a couple more extensions. Happy to take on some of that work but I get the impression that 'real' development work is happening in a private version of this repo somewhere and this is more of a public dump than an active codebase. There was a series of releases in April, none since and the PGVector PR seems to be stagnant. While obviously nobody owes anyone anything with open source would it be possible to get some kind of sense of what's happening here and where development is headed? Would love to be able to both use this, it looks like phenomenal work - and be able to effectively contribute back.

**Comment by samwillis** [comment]
Hey @alexgraul,

Yep, there was a period where we went back and rebuilt the tooling for PGlite, but it's all now being developed in the main repo under the `next` branch: https://github.com/electric-sql/pglite/pull/112

I've been away on vacation the last few weeks, but the intention is to get `next` merged into main early next week and start publishing some alphas of the new build to npm, ahead of a release in a few weeks time.

We are not going to go back to having any development in a private repo or fork, that went on too long, was longer than intended, and a mistake.

You can see a list of some of what going to be in the next release in #112, but it will also include some of the other PRs that are open (#118, #104), and we are going to publish an example repo of how to build custom extensions.

I'm very keen for PGlite to be a community project, open to contributions.

**Comment by alexgraul** [comment]
Hey Sam, that's awesome to hear and thankyou for the clarity, will look forward to having a play once that release is in alpha!

**Reply by samwillis** [reply]
Hey @alexgraul 

The Alpha is now out if you want to give it a go.

We are aiming for a final release in about two weeks time.

**Reply by alexgraul** [reply]
Awesome, going to do so right now!
